common/test/acceptance/pages/common/utils.py

```diff
@@ -56,10 +56,6 @@ def confirm_prompt(page, cancel=False, require_notification=None):
     cancel is True.
     """
     page.wait_for_element_visibility('.prompt', 'Prompt is visible')
-    page.wait_for_element_visibility(
```

Comment:
This check was added by Christine in this PR-- #9506. Her PR added unit tests as well, which as far as I know are not flaky. But this test in the Bok Choy tests does appear to be problematic; I get suspicious of focus tests in Bok Choy, as the tests run very quickly. I can't guarantee that the functionality itself is not flaky, but it seemed to work fine when I tested manually.